core-bundle/src/EventListener/CsrfTokenCookieListener.php

```diff
@@ -86,15 +131,20 @@ private function getTokensFromCookies(ParameterBag $cookies): array
         $tokens = [];
 
         foreach ($cookies as $key => $value) {
-            if (!\is_string($key)) {
-                continue;
-            }
-
-            if (0 === strpos($key, $this->cookiePrefix) && preg_match('/^[a-z0-9_-]+$/i', $value)) {
+            if ($this->isCsrfCookie($key, $value)) {
                 $tokens[substr($key, \strlen($this->cookiePrefix))] = $value;
             }
         }
 
         return $tokens;
     }
+
+    private function isCsrfCookie($key, string $value): bool
+    {
+        if (!\is_string($key)) {
+            return false;
+        }
+
+        return 0 === strpos($key, $this->cookiePrefix) && preg_match('/^[a-z0-9_-]+$/i', $value);
```

Comment:
Just double checking: Is it correct we use the `i` modifier here?

Reply:
Yes